src/ocp_postprocess.rs

```diff
@@ -218,6 +247,107 @@ pub(crate) async fn fix_olm_secret_hash_annotation(in_memory_etcd_client: &Arc<I
     Ok(())
 }
 
+fn verbosity(log_level: &str) -> String {
+    match log_level {
```

Comment:
Add comment mentioning this follows https://github.com/openshift/library-go/blob/15d11d2f6bfcbe15679acd184ac69c77aa2e65bc/pkg/operator/loglevel/util.go#L13-L27